app/helpers/pre_built_pages_helper.rb

```diff
@@ -1,14 +1,15 @@
 module PreBuiltPagesHelper
-  def include_page_content(path, default = nil)
+  def include_page_content(path, default = nil, postfix = "")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.